resources/install.conf

```diff
@@ -10,5 +10,5 @@ Which disk is the root disk = sd0
 Use (W)hole disk MBR, whole disk (G)PT or (E)dit = g
 URL to autopartitioning template for disklabel = {{ DISKLABEL_TEMPLATE }}
 Http Server = cdn.openbsd.org
-Set name(s) = -game* -x* -man*
```

Comment:
Please move this and related changes to a separate PR, as discussed in the issue.